serverless/internal/storage/fs/fs.go

```diff
@@ -246,12 +247,76 @@ func (fs *Storage) ScanSequenced(begin uint64, f func(seq uint64, entry []byte)
 // If no complete tile exists at that location, it will attempt to find a
 // partial tile for the given tree size at that location.
 func (fs *Storage) GetTile(level, index, logSize uint64) (*api.Tile, error) {
-	return nil, errors.New("unimplemented")
+	tileSize := storage.PartialTileSize(level, index, logSize)
+	p := filepath.Join(layout.TilePath(fs.rootDir, level, index, tileSize))
+	t, err := ioutil.ReadFile(p)
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		return nil, fmt.Errorf("failed to read tile at %q: %w", p, err)
+	}
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	if err != nil {
		if !errors.Is(err, os.ErrNotExist) {
			return nil, fmt.Errorf("failed to read tile at %q: %w", p, err)
	  	}
	 	return nil, err
	}
```

nit suggestion: It reads a little clearer to me if both errors are handled in the same block.

Reply:
done